pymeasure/instruments/instrument.py

```diff
@@ -599,3 +647,73 @@ def check_errors(self):
             return errors
         else:
             raise NotImplementedError("Non SCPI instruments require implementation in subclasses")
+
+
+class Channel(ChannelBase):
+    """The base class for channel definitions.
+
+    This class supports dynamic properties like :class:`Instrument`,
+    but requires an instrument as a parent for communication.
+    The default implementation of :meth:`write` uses :code:`str.format` to
+    exchange :code:`'{ch}'` for the channel name.
+
+    :param instrument: The instrument (an instance of :class:`~pymeasure.instruments.Instrument`)
+        to which the channel belongs.
+    :param name: Name of the channel, as it is used for the communication.
```

Comment:
What do you think about calling `name` `id`, instead? This makes it a little more different from the similar class `Instrument`, which also has a `name` argument, and thus reduces the possibility for confusion.